server/py/services/api/api/endpoints/model_endpoints.py

```diff
@@ -229,7 +229,7 @@ async def list_model_endpoints(
     start: Optional[datetime] = None,
     end: Optional[datetime] = None,
     top_level: bool = Query(False, alias="top-level"),
-    tsdb_metrics: bool = True,
+    tsdb_metrics: Optional[str] = None,
```

Comment:
```suggestion
    tsdb_metrics: Optional[str] = Query(None, alias="tsdb-metrics"),
```
The name of the parameter is invalid. You should not use underscores.